.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -8,106 +8,45 @@ jobs:
     vmImage: vs2017-win2016
```

Comment:
Had suggested `vs2017-win2016` as that is what we used before (please see above)

That said, it looks like [Azure deprecated this image]( https://devblogs.microsoft.com/devops/hosted-pipelines-image-deprecation/ )